main/src/main.rs

```diff
@@ -266,6 +269,45 @@ struct TraceArgs {
     use_native_tracer: bool,
 }
 
+#[derive(Args, Clone, Debug)]
+pub struct SimulateArgs {
+    /// RPC endpoint.
+    #[arg(short, long, default_value = "http://localhost:8547")]
+    endpoint: String,
+
+    /// From address.
+    #[arg(short, long)]
+    from: Option<H160>,
+
+    /// To address.
+    #[arg(short, long)]
+    to: Option<H160>,
+
+    /// Gas limit.
+    #[arg(long)]
+    gas: Option<u64>,
+
+    /// Gas price.
+    #[arg(long)]
+    gas_price: Option<U256>,
+
+    /// Value to send with the transaction.
+    #[arg(short, long)]
+    value: Option<U256>,
+
+    /// Data to send with the transaction, as a hex string (with or without '0x' prefix).
+    #[arg(short, long)]
+    data: Option<String>,
```

Comment:
https://docs.rs/ethers-core/latest/ethers_core/types/struct.Bytes.html should be a better option here

Reply:
Done! 
Thanks Raul!